src/page/user-profile-demo.js

```diff
@@ -0,0 +1,371 @@
+import React from 'react';
+import {Row, Col} from 'react-flex-proto';
+import {ProgressBar, Input, Button, Switch, Panel, Select, Modal, Page, Breadcrumbs} from 'react-blur-admin';
+import {Link} from 'react-router';
+
+export class UserProfileDemo extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      switches: [true, false, false, true, true, false],
+      addLinkModal: false,
+    };
+  }
+
+  onSwitchChange(index) {
+    let switches = this.state.switches;
```

Comment:
We should use `_.assign` so we don't modify the `this.state.switches` object directly. It should still technically work as-is but it's generally not recommended.